.travis.yml

```diff
@@ -1,8 +1,29 @@
 language: python
-env:
-- TOXENV=py27
-- TOXENV=py36
+
+notifications:
+  email: false
+
+python:
+   - "3.6"
+   - "3.7"
+   - "3.8"
+
+matrix:
+  allow_failures:
+    - python: 3.8
+
 install: pip install tox
 script: tox
 services:
   - redis-server
+
+deploy:
+  provider: pypi
+  user: inveniosoftware
+  password:
+    secure: "jhM7LIYWXbVPrdm2c+by+HZGJ6RrLDcaWGbnVt0foFvZRmrfq79GZqVjAaPerUqq/FLX6TggRAU1U863CszhlaNxpBB2W1Pgn8aMYA+O084IfR8YZvDzcp473djMBvTWsSX63FWXwVNcDwuRvwWDTiecz+3V4aDzLu99wPV5dERUYMKzU/n/4QdwWJnGXeNMLGBQc+t966Fyg3xPs7OIldWk0tkVucAwuZVYFSL/L8OeVIj6x76JmhK/my+cxP9eStui+btJWooY0hZQ0A1J3o7RMc0sywDO0xMeUEth7tjeIOj+id03o6MuU4jTbEeYhjGp5sdQNweWq9QN7BRugjr3BR0MVeml7xlBGoJjsm5u/FNfSwOzYKHzF2S3ApC0mJCiZdZB+/bvUjj2fID8mZ7PrKy6P1UnlxLveTU07eoGi/fTz6GJ0tAB20/vm+OVdv8/1PaTUhwfh0gBp4lVC/j/7N3DdyMG87+5ZOQ26Xc43KTDMQmn5qNCpdEC1IyV8ubwgtjzen7Pbu2pL4/STwtfUpGhx3LS0ugAFXwAQkdAj23f1oShQ2WiYuoA3L2HWkX5YdGYGcQFaokztschAlxGEPjjWxBh2W6u1hl6/ikfCMognnHyQEBzneRkFJFeIgpUkXs53wd7hcR1uTO+9vh5jG0idtdbknvK/ScVsYI="
+  distributions: "compile_catalog sdist bdist_wheel"
+  on:
+    tags: true
+    repo: inveniosoftware/Flask-KVSession-Invenio
```

Comment:
Not sure if this is enough to define the final package name. I thought this was something that needs a change in setup.py ([see Flask-Security-Fork](https://github.com/inveniosoftware/flask-security-fork/blob/cb2b3bb080dde4ec7f207522ca7d901e095e5b84/setup.py#L61)).

Also we need to rename the GitHub repository to Flask-KVSession-Invenio for this to work :)

Reply:
Name comes from setup.py. Therefore, there is no need for the repo name here and no need to change it. Fixed :) Thanks for the catch!